doc/setup_instructions.md

```diff
@@ -521,13 +521,18 @@ the pre-commit hook section for how to speed things up.
 Pre-commit hook
 ---------------
 
-On Linux you can enable a pre-commit hook to automatically run the
+You can enable a pre-commit hook to automatically run the
 formatting checks before each commit to avoid accidentally committing
-code with formatting issues. To install the hook run the following
-script:
+code with formatting issues.
+
+To install pre-commit use the commands for Windows `pip install pre-commit`,
+if you are using Linux you can install with your package manager.
```

Comment:
I just installed pre-commit with "sudo pip install pre-commit" and it works perfectly fine. This line should be deleted.